direct/src/cluster/ClusterClient.py

```diff
@@ -145,7 +145,7 @@ def controlObjectTask(self, task):
 
     def sendNamedMovementDone(self, serverList = None):
 
-        if (serverList == None):
+        if (serverList is None):
```

Comment:
While you're at it making style changes, perhaps you could also remove the unnecessary parentheses in `if` statements like this one.